src/typescript/02-ide-support/ide-support.md

```diff
@@ -24,3 +24,9 @@ Integrated Development Environments have come a long way in supporting modern we
 <a href="https://www.jetbrains.com/webstorm/download/">Webstorm</a> is a platform by JetBrains that is loved for its great code refactoring assistance and version control integration, but it does require a paid subscription.
 
 <img src="../static/img/webstorm-screenshot.png" width="100%" alt="Webstorm screenshot" />
+
+### StackBlitz
+
+<a href="https://stackblitz.com/">StackBlitz</a> is an instant fullstack web IDE for the JavaScript ecosystem. It's powered by WebContainers, the first WebAssembly-based operating system which boots the Node.js environment in milliseconds, securely within your browser tab.
+
+<img src="../static/img/stackblitz-screenshot.png" width="100%" alt="StackBlitz screenshot" />
```

Comment:
This screenshot should show the TypeScript code, not PMO. Same thing for the Webstorm screenshot.

I’d also update the `alt` text for all of these to describe what is being pointed out in the screenshots.

Here’s an example of what could be written for the current VS Code screenshot:

> Screenshot of a TypeScript file in VS Code. When the name of an interface is hovered over, VS Code displays a popover with the interface’s implementation so it’s easier to see what is in the interface.